File: c#_SDK/PayHubSDK/PayHubSDK/com/payhub/ws/model/EncryptedTrackData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace PayHubSDK.com.payhub.ws.model
{
    ///  @author Agustin Breit 
    ///   
    [DataContract]
    public class EncryptedTrackData
    {
        [DataMember]
        private string encrypted_track = "";
        [DataMember]
        private string swiper_brand = "IDTECH";
        [DataMember]
        private string swiper_model = "UNIMAGII";
        

        public EncryptedTrackData(string value)
        {
            // TODO: Complete member initialization
            this.encrypted_track = value;
        }

    }
}

File: ruby_SDK/PayHubSDK/lib/PayHubSDK/com/payhub/ws/model/encrypted_track_data.rb
class EncryptedTrackData
  include JsonSerializer
  ATTRS=[:encrypted_track,:swiper_band,:swiper_model]
  attr_accessor *ATTRS
  @encrypted_track=""
  @swiper_band="IDTECH"
  @swiper_model="UNIMAGII"

  def initialize

  end
end